### DISCUSSION TITLE
Any way to change the favicon to the company, rather than sveltia?

### DISCUSSION THREAD

**Discussion by rchrdnsh** [sveltia/sveltia-cms#163] [Q&A] [answered]
The docs page is rather long and I have not found this yet, but can I use the company logo for the favicon as well?

Thanks :-)

**Comment by kyoshino** [comment, marked as answer]
Haven’t checked Decap, but I think [`logo_url`](https://decapcms.org/docs/configuration-options/#custom-logo) should override the default Sveltia favicon, just like the log-in page logo. I’m making the change now.

**Comment by kyoshino** [comment]
The fix landed in [v0.34.2](https://github.com/sveltia/sveltia-cms/releases/tag/v0.34.2)!

**Reply by rchrdnsh** [reply]
You are so fast!
